packages/react/src/components/InAppMessaging/BannerMessage/BannerMessage.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button } from '../../../primitives/Button';
+import { Flex } from '../../../primitives/Flex';
+import { Heading } from '../../../primitives/Heading';
+import { Image } from '../../../primitives/Image';
+import { Text } from '../../../primitives/Text';
+import { CloseIconButton } from '../CloseIconButton';
+import { useThemeBreakpoint } from '../../../hooks/useThemeBreakpoint';
+import { useMessageProps } from '../hooks/useMessageProps';
+
+export default function BannerMessage({
+  alignment = 'right',
+  body,
+  container,
+  header,
+  image,
+  layout,
+  onClose,
+  onDisplay,
+  position = 'top',
+  primaryButton,
+  secondaryButton,
+}): JSX.Element | null {
+  const breakpoint = useThemeBreakpoint();
+
+  const messageProps = useMessageProps({
+    body,
+    container,
+    header,
+    image,
+    layout,
+    onClose,
+    onDisplay,
+    primaryButton,
+    secondaryButton,
+  });
+
+  const {
+    hasPrimaryButton,
+    hasSecondaryButton,
+    hasRenderableImage,
+    shouldRenderMessage,
+  } = messageProps;
+
+  if (!shouldRenderMessage) {
+    return null;
+  }
+
+  return (
+    <Flex
+      className={`amplify-in-app-messaging-banner__container amplify-in-app-messaging-banner-${position} amplify-in-app-messaging-banner-${alignment} ${String(
+        breakpoint
```

Comment:
Is `breakpoint` not a `string` by default?

Reply:
Yeah, it is. VSC was giving me errors, but it's now fine. Fixing.